src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Authoring.Tests/Microsoft.Azure.CognitiveServices.LUIS.Authoring.Tests.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
   <PropertyGroup>
-    <Description>Microsoft.Azure.CognitiveServices.LUIS-Programmatic.Tests Class Library</Description>
-    <AssemblyName>Microsoft.Azure.CognitiveServices.LUIS-Programmatic.Tests</AssemblyName>
-    <Version>2.0.0</Version>
+    <Description>Microsoft.Azure.CognitiveServices.LUIS-Authoring.Tests Class Library</Description>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.LUIS-Authoring.Tests</AssemblyName>
+    <Version>0.9.0-preview</Version>
```

Comment:
These tests are never published, simply set the version to `1.0.0` and never update them